src/election.jl

```diff
@@ -15,12 +15,12 @@ function Election(name::String, parties::Array{String, 1}, num_districts::Int)
     return Election(name, parties, vote_counts, vote_shares)
 end
 
-function count_votes(name::String, election::Election)::DistrictScore
```

Comment:
This function was doing triple duty! Updating the `Election` object and returning the vote counts and vote shares for all parties in the district.